src/components/dialogs/create-study-dialog.js

```diff
@@ -137,7 +136,7 @@ const SelectCase = () => {
  * @param {Boolean} open Is the dialog open ?
  * @param {EventListener} onClose Event to close the dialog
  */
-export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
+export const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
```

Comment:
`fromExistingCase` ?
Same as  back end

Reply:
Where is it in the backend ?
It's not a boolean so I don't think using the prefix "from" is appropriate...